packages/autocomplete-plugin-algolia-insights/src/createAlgoliaInsightsPlugin.ts

```diff
@@ -7,6 +7,7 @@ import {
   noop,
   safelyRunOnBrowser,
 } from '@algolia/autocomplete-shared';
+import { AutocompleteReshapeSource } from '@algolia/autocomplete-shared/dist/esm/core';
```

Comment:
Nitpick: should we reexport it publicly from **@algolia/autocomplete-shared** ?

Reply:
it would be nice, but there's other instances of importing from dist/esm so that should all be done together :)